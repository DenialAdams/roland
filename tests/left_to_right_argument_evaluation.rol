proc eval(foo: String) -> bool {
   print(foo);
   return true;
}

proc aux(x: bool, y: bool, named a: bool, named b: bool) {
   print("done");
}

proc main() {
   aux(eval("x"), eval("y"), b: eval("b"), a: eval("a"));
}
