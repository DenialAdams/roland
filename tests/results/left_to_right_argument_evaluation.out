x
y
b
a
done
